src/app/Main/Body/QueryPage/Results/TableResults/TableResults-Options.component.tsx

```diff
@@ -57,22 +72,30 @@ const BlueRadio = withStyles(() =>
  * @returns {JSX.Element} - option bar component
  */
 export function ResultsTableOptionsComponent(props: {
+    isSumRowCol: boolean;
     tableOption: DisplayOptionType;
     onRadioChange: (eventTargetValue: string) => void;
+    onCheckboxChange: (checked: boolean) => void;

```

Comment:
As stated in the last review, try to use more context sensitive names: E.g. this could be: onShowSumsToggle: (showSums: boolean)

Reply:
Done. Commit "Renaming"